### PR TITLE
2017 Chevy Malibu

### PR DESCRIPTION
**Pull request by tylergets**
Currently using all settings from the Chevy Volt, will update with more precise measurements when I am able to get them.

### CODE REVIEW

File: selfdrive/car/gm/values.py [hunk 1]
```diff
@@ -14,7 +15,7 @@ class CruiseButtons:
 
 def is_eps_status_ok(eps_status, car_fingerprint):
   valid_eps_status = []
-  if car_fingerprint == CAR.VOLT:
+  if car_fingerprint == in (CAR.VOLT, CAR.MALIBU):
```
Comment (energee) [inline selfdrive/car/gm/values.py]: Should be `if car_fingerprint in (CAR.VOLT, CAR.MALIBU)`

**Comment by tylergets** [comment]
@rbiasini it's working without fault, both long + lat need _some_ work, however. I am not sure what work that is being done for the general GM tuning in #326 applies here, but I'd be glad to test. I will get a proper cabana drive posted here over the weekend.

Some initial observations, the car seems to brake unnecessarily hard in certain situations, definitely the opposite of smooth. The lkas frequency/step might be off as well.

**Comment by rbiasini** [comment]
@tylergets , is longitudinal tuning still not ok?

**Comment by tylergets** [comment]
@rbiasini the current gm tune is a bit aggressive, this applies for the currently supported chevy volt as well. Tuning by @jamezz-comma in #326 and his branches greatly improves this. But because this is no different than the volt, I don't see what stops this from being good to go. 

I have also updated interface.py to include the malibu specific curb weight and wheelbase.

**Comment by Jamezz** [comment]
As long as the car stops safely I don't see why this can't be merged. The volt longitudinal changes should help a lot with all GM.

**Comment by rbiasini** [comment]
@tylergets ok, great. Will merge this for now. GM longitudinal changes from @Jamezz later.

**Comment by rbiasini** [comment]
@tylergets I don't see any uploaded log from any MALIBU. Can you please DM me a cabana route?

**Comment by rbiasini** [comment]
@tylergets merged internally. It will be included in 0.5.6

**Comment by tylergets** [comment]
Thanks comma team!

**Pull request closed without merging**